terracumber_config/tf_files/SUSEManager-4.2-beta-build-validation.tf

```diff
@@ -755,6 +757,59 @@ module "ubuntu1604-minion" {
 
 }
 
+// TODO: When we enable debian in sumaform enable these
+/*
+module "debian9-minion" {
+  providers = {
+    libvirt = libvirt.florina
+  }
+  source             = "./modules/minion"
+  base_configuration = module.base_newsle_ubuntu.configuration
```

Comment:
Call me sheldon, but maybe on the long run this configuration should be renamed with `debian` instead of `ubuntu` (ubuntu is a kind of debian, reverse is not true).

Reply:
agreed in principle for this. For now I will suggest we do this in a separate PR for all tf files that we do this. It's out of scope for this current PR